src/service/http-server.integration.test.ts

```diff
@@ -0,0 +1,232 @@
+import * as fs from 'fs';
+import * as jwt from 'jsonwebtoken';
+import * as path from 'path';
+import * as request from 'supertest';
+import * as pixelmatch from 'pixelmatch';
+import * as fastPng from 'fast-png';
+
+import { HttpServer } from './http-server';
+import { ConsoleLogger } from '../logger';
+import { ServiceConfig } from './config';
+import { createSanitizer } from '../sanitizer/Sanitizer';
+
+const dashboardUid = 'd10881ec-0d35-4909-8de7-6ab563a9ab29';
+const panelIds = {
+  graph: 1,
+  table: 2,
+  error: 3,
+  slow: 4,
+};
+const renderKey = jwt.sign(
+  {
+    renderUser: {
+      org_id: 1,
+      user_id: 1,
+      org_role: 'Admin',
+    },
+  },
+  '-',
+  { algorithm: 'HS512' }
+);
+
+const goldenFilesFolder = './src/testdata';
+const serviceConfig: ServiceConfig = {
+  service: {
+    host: undefined,
+    port: 8081,
+    metrics: {
+      enabled: false,
+      collectDefaultMetrics: true,
+      requestDurationBuckets: [0.5, 1, 3, 5, 7, 10, 20, 30, 60],
+    },
+    logging: {
+      level: 'debug',
+      console: {
+        json: true,
+        colorize: false,
+      },
+    },
+    security: {
+      authToken: '-',
+    },
+  },
+  rendering: {
+    args: ['--no-sandbox', '--disable-gpu'],
+    ignoresHttpsErrors: false,
+    timezone: 'Europe/Paris',
+    width: 500,
+    height: 300,
+    deviceScaleFactor: 1,
+    maxWidth: 1000,
+    maxHeight: 500,
+    maxDeviceScaleFactor: 2,
+    pageZoomLevel: 1,
+    mode: 'default',
+    clustering: {
+      monitor: false,
+      mode: 'browser',
+      maxConcurrency: 5,
+      timeout: 30,
+    },
+    verboseLogging: true,
```

Comment:
The successful output is quite verbose. I tried changing this value to false, but it still logs a lot of debug messages. Is it possible to keep it less verbose by default?

Reply:
Yes I updated the config to remove some logs.